src/NHibernate/Cache/ICacheConcurrencyStrategy.cs

```diff
@@ -139,4 +140,84 @@ public partial interface ICacheConcurrencyStrategy
 		/// <value>The <see cref="ICache"/> for this strategy to use.</value>
 		ICache Cache { get; set; }
 	}
-}
\ No newline at end of file
+
+	internal static partial class CacheConcurrencyStrategyExtensions
+	{
+		/// <summary>
+		/// Attempt to retrieve multiple objects from the Cache
+		/// </summary>
+		/// <param name="cache">The cache concurrency strategy.</param>
+		/// <param name="keys">The keys (id) of the objects to get out of the Cache.</param>
+		/// <param name="txTimestamp">A timestamp prior to the transaction start time</param>
+		/// <returns>An array of cached objects or <see langword="null" /></returns>
+		/// <exception cref="CacheException"></exception>
+		//6.0 TODO: Merge into ICacheConcurrencyStrategy.
+		public static object[] GetMultiple(this ICacheConcurrencyStrategy cache, CacheKey[] keys, long txTimestamp)
```

Comment:
I would like to have `interface IMultiCacheConcurrencyStrategy : ICacheConcurrencyStrategy` interface instead.

Reply:
I've added a `IBatchableCacheConcurrencyStrategy` interface.